echopype/echodata/sensor_ep_version_mapping/v05x_to_v06x.py

```diff
@@ -0,0 +1,1131 @@
+    1. Moves ``tilt_x/y(ping_time)`` to the `Platform` group.
+    2. Moves ``temperature_counts(ping_time)``,
+    ``tilt_x/y_count(ping_time)``, ``DS(channel)``, ``EL(channel)``,
+    ``TVR(channel)``, ``VTX(channel)``, ``Sv_offset(channel)``,
+    ``number_of_samples_digitized_per_pings(channel)``,
+    ``number_of_digitized_samples_averaged_per_pings(channel)``
+    to the `Vendor` group:
+    3. Removes the variable `cos_tilt_mag(ping_time)`
+    4. Moves the following attributes to the ``Vendor`` group:
+    ``tilt_X_a/b/c/d``, ``tilt_Y_a/b/c/d``,
+    ``temperature_ka/kb/kc/A/B/C``, ``number_of_frequency``,
+    ``number_of_pings_per_burst``, ``average_burst_pings_flag``
+
+    Parameters
+    ----------
+    ed_obj : EchoData
+        EchoData object that was created using echopype version 0.5.x.
+    sensor : str
+        Variable specifying the sensor that created the file.
+
+    Notes
+    -----
+    The function directly modifies the input EchoData object.
+    """
+
+    if sensor == "AZFP":
+
+        beam_to_plat_vars = ["tilt_x", "tilt_y"]
+
+        for var_name in beam_to_plat_vars:
+            ed_obj["Platform"][var_name] = ed_obj["Sonar/Beam_group1"][var_name]
+
+        beam_to_vendor_vars = [
+            "temperature_counts",
+            "tilt_x_count",
+            "tilt_y_count",
+            "DS",
+            "EL",
+            "TVR",
+            "VTX",
+            "Sv_offset",
+            "number_of_samples_digitized_per_pings",
+            "number_of_digitized_samples_averaged_per_pings",
+        ]
+
+        for var_name in beam_to_vendor_vars:
+            ed_obj["Vendor"][var_name] = ed_obj["Sonar/Beam_group1"][var_name]
+
+        beam_to_vendor_attrs = ed_obj["Sonar/Beam_group1"].attrs.copy()
+        del beam_to_vendor_attrs["beam_mode"]
+        del beam_to_vendor_attrs["conversion_equation_t"]
+
+        for key, val in beam_to_vendor_attrs.items():
+
+            ed_obj["Vendor"].attrs[key] = val
+            del ed_obj["Sonar/Beam_group1"].attrs[key]
+
+        ed_obj["Sonar/Beam_group1"] = ed_obj["Sonar/Beam_group1"].drop(
+            ["cos_tilt_mag"] + beam_to_plat_vars + beam_to_vendor_vars
+        )
+
+
+def _rename_mru_time_location_time(ed_obj):
+    """
+    Renames location_time to time1 and mru_time to
+    time2 wherever it occurs.
+
+    Parameters
+    ----------
+    ed_obj : EchoData
+        EchoData object that was created using echopype version 0.5.x.
+
+    Notes
+    -----
+    The function directly modifies the input EchoData object.
+    """
+
+    for grp_path in ed_obj.group_paths:
+
+        if "location_time" in list(ed_obj[grp_path].coords):
+
+            # renames location_time to time1
+            ed_obj[grp_path] = ed_obj[grp_path].rename(name_dict={"location_time": "time1"})
+
+        if "mru_time" in list(ed_obj[grp_path].coords):
+
+            # renames mru_time to time2
+            ed_obj[grp_path] = ed_obj[grp_path].rename(name_dict={"mru_time": "time2"})
```

Comment:
Seems that we need to attach these time coordinates with the nice set of v0.6.0 attributes also?

Reply:
I see that you have another 2 functions `_add_time_comment_in_environment` and `_add_time_comment_in_platform` for this, but if I am not mistaken, those only deal the "comment" attribute. From #682 it seems `long_name` needs update too, so may be a good idea to just copy the entire new set, perhaps in the functions that you already have?

Reply:
We have not arrived at a consensus for #682 yet. Once we have, I will implement it. I planned on putting the changes in the functions you mention and renaming them (I haven't fully thought this out though).